host-ext/nacl_src/kiwi.cc

```diff
@@ -257,7 +267,7 @@ class KiwiInstance : public pp::Instance {
 
         if (server_version_ != VERSION) {
             /* TODO: Remove VF1 compatiblity */
-            if (server_version_ == "VF1") {
+            if (server_version_ == "VF1" || server_version_ == "VF2") {
```

Comment:
Actually, can we not pop up the warning for the update?  Most people probably won't care in this case...